### ISSUE TITLE
Feature request: Status icons (for conference feedbacks)?

### ISSUE THREAD

**Issue by r4dh4l** [mumble-voip/mumble#4102]
Hi,

would it be possible to let users change the user icon to provide instant status feedbacks like

- "I can not here you"
- "I'm confused"
- "I'm bored"
- "I need help"
- "I'm convinced (by what you say)"
- "I'm sceptical"
- "I agree/disagree/have no opinion about that"
- "I have a question"

?

I know that you could create channels for such feedbacks and let the users just join the channels but the idea is: Giving a lecture the users can directly indicate what the lecturer would normally see by facial expressions.

Thx for Mumble!

**Comment by Krzmbrzl** [comment]
Tbh I don't think that these icons would be used. The key aspect about facial expressions in this context is that they happen without the person having to explicitly think about it. With these icons on the other hand the user would have to take an explicit action which I don't think many will do.

Furthermore these icon.-statuses ought to be incomplete. You'll always find someone who wants to have an icon for yet another status.
And finally what happens if you set your icon to e.g. "I'm confused" but the lecturer keeps talking and then you also want to signal e.g. that you agree with what was said. In order to allow for these situations we'd have to allow arbitrary many icons at a time which would quickly clutter the UI.

TL;DR I don't think that it is practical to implement this (properly) and most importantly: I doubt that people would actually make use of this kind of feature.

**Comment by r4dh4l** [comment]
> Tbh I don't think that these icons would be used.

Well, today I gave alecture where students requested exactly this and I rembered that BigBlueButton (which is to oversized for the most conference cases in my opinion) has such a feature. Here is a screenshot:

![Screenshot_2020-04-24_21-22-27](https://user-images.githubusercontent.com/44623983/80249583-68bf9080-8672-11ea-827a-46bfb0f57ccc.png)

**Comment by Krzmbrzl** [comment]
> the students requested exactly such a feature

Okay then. Wouldn't have thought that. Thanks for the clarification :ok_hand: :

**Comment by streaps** [comment]
This might useful in some scenarios, but for other types of conversations it is an anti-feature. There is also no universal set of status icons that works for every situation.

If this feature will be added, an option to disable it by default would be nice (server wide or ACL).

**Comment by Krzmbrzl** [comment]
>  but for other types of conversations it is an anti-feature.

What scenario do you have in mind? I mean it's only an icon after all. If you are not interested in it, you can ignore it, can't you?

Adding an ACL for this won't hurt in any case though anyways :)

**Comment by bendem** [comment]
To handle large meetings, the hand raising status is very useful to allow someone to lead discussions and distribute the virtual mic. It's less of an issue with mumble because low latency, but would still be a welcomed addition.
